mindocr/data/builder.py

```diff
@@ -164,6 +164,7 @@ def build_dataset(
     is_main_device = device_id == 0
     _logger.info(
         f"Creating dataloader (training={is_train}) for device {device_id}. Number of data samples: {num_samples}"
+        f" per device ({num_samples * num_devices} total)."
```

Comment:
I am not sure if it is evenly distributed if the total num is not divisible by device num?

Reply:
MindSpore pads datasets to have the same number of samples on each device. For example, a dataset with 25 samples, distributed on 2 devices with batch size of 10:
1st device log
```shell
mindocr.data.builder INFO - Creating dataloader (training=True) for rank 0. Number of data samples: 13
mindocr.data.builder INFO - Creating dataloader (training=False) for rank 0. Number of data samples: 13
```
2nd device log
```shell
mindocr.data.builder INFO - Creating dataloader (training=True) for rank 1. Number of data samples: 13
mindocr.data.builder INFO - Creating dataloader (training=False) for rank 1. Number of data samples: 13
```